### PR TITLE
Extend container with `command` and `args` fields.

### PR DESCRIPTION
**Pull request by heshamMassoud**
1. Add `command` and `args` field to container resource. 
2. Add [documentation](https://github.com/commercetools/k8s-charts/pull/7/files#diff-8e08fd01be5542564dd7fb0c96484c92) for new configurable value.
2. [Bump chart version](https://github.com/commercetools/k8s-charts/commit/757eaa194fb9d3ad08401f21d6f43ee9d1e3b4ed).

### CODE REVIEW

File: charts/cronjob/values.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ image:
   repository: 'commercetools/commercetools-email-retry-processor'
   tag: '1.0.0'
   pullPolicy: 'IfNotPresent'
+  command: ['cmd1', 'cmd2']
```
Comment (lojzatran) [inline charts/cronjob/values.yaml]: What about put there a `hello world` cmd?

**Review by lojzatran — APPROVED** [review]

File: charts/cronjob/values.yaml (hunk 1 above)
Comment (heshamMassoud) [reply charts/cronjob/values.yaml]: don't you think `'cmd1', 'cmd2'` are expressive enough?

Comment (lojzatran) [reply charts/cronjob/values.yaml]: They just don't work, but other than that, it's ok.

Comment (heshamMassoud) [reply charts/cronjob/values.yaml]: Agreed. @svenmueller mentioned before that 
> charts should be able to run out-of-the-box with default values.

 So in this case, I left it out empty and added a comment to describe it next to it. WDYT @lojzatran ? 8f1a05e

Comment (lojzatran) [reply charts/cronjob/values.yaml]: Ok to me, leave it empty then. 👍

**Review by svenmueller — APPROVED** [review]

**Review by tetianakravchenko — APPROVED** [review]

**Pull request merged**